libs/designsystem/testing-jest/src/lib/kirby-testing.module.ts

```diff
@@ -10,7 +10,6 @@ export function modalControllerFactory() {
     showModal: jest.fn(),
```

Comment:
If we are discontinuing jest, these should properly be adjusted

Reply:
Agree, but let's leave it for now, other projects might consider using jest